roles/barbican/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: Check if barbican user exists
+  action: shell getent passwd barbican
+  register: barbican_user
+  failed_when: False
+  changed_when: False
+
+- name: barbican user
+  user: name=barbican shell=/bin/false createhome=no
+  when: barbican_user|success
+
```

Comment:
I think it's easier to read the dictionaries if we don't put it inline:
```
name: barbican user
  user:
    name: barbican
    shell: /bin/false
    createhome: no
  when: barbican_user|success
```
I think all the items in this PR should follow that convention.